source/stylesheets/base/_common.scss

```diff
@@ -30,3 +30,22 @@ nav ul {
   position: absolute !important;
   width: 1px;
 }
+
+.quest-issue {
+  text-align: center;
+
+  h1 {
+    margin-top: 2.5em;
```

Comment:
Should try to stick to the `$space-` vars. `$space-med-sm` is `3rem` so it should look okay here. Although I'd rather use padding-top than margin for this section since that's really what this is accomplishing.